sda-commons-server-auth-testing/src/main/java/org/sdase/commons/server/auth/testing/AuthClassExtension.java

```diff
@@ -14,7 +14,7 @@
 public class AuthClassExtension extends AbstractAuth
     implements BeforeAllCallback, AfterAllCallback {
 
-  private static final String AUTH_RULE_ENV_KEY = "AUTH_RULE";
+  public static final String AUTH_RULE_ENV_KEY = "AUTH_RULE";
```

Comment:
While we're at it: Is this naming still appropriate (rule -> extension)?

Reply:
I think it's a good opportunity to rename the variable name + value. Consequences for customers: Double check if you use that variable in any one of your `test-config.yaml`s and rename it accordingly.